content/docs/04-clusters/06-cluster-management/02.5-certificate-management.md

```diff
@@ -0,0 +1,421 @@
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+Palette installs Kubernetes through the tool, [kubeadm](https://kubernetes.io/docs/reference/setup-tools/kubeadm). As a result, all deployed clusters include auto-generated Public Key Infrastructure (PKI) certificates created by kubeadm. We recommend you review the [PKI certificates and requirement](https://kubernetes.io/docs/setup/best-practices/certificates) documentation from the official Kubernetes documentation to learn more about the auto-generated certificates and to understand their purpose better.
+
+This reference page focuses on how to renew the PKI certificates through Palette. You have two options for how you can renew the cluster PKI certificates:
+
+  <br />
+
+  * Automatic Certificate Renewal
+
+
+  * Manual Certificate Renewal
+
+
+
+<InfoBox>
+
+Certificates created by kubeadm expire after 365 days. The Root Certificate Authority (CA) is valid for 3652 days or ten years.
+
+</InfoBox>
+
+
+You can learn more about each option in the following sections. 
+
+
+
+# Automatic Certificate Renewal
+
+When you upgrade the control plane on a cluster, the PKI certificates are automatically updated. Upgrading a cluster's Kubernetes version, whether a minor patch or a major release, results in renewed PKI certificates. This is the method that requires the least user actions when it comes to renewing PKI certificates. We recommend regularly updating your clusters to stay up-to-date with security fixes and best practices. By keeping your host cluster up-to-date, you prevent the scenario of PKI certificates from reaching their one-year expiration date.
+
+<br />
+
+<InfoBox>
+
+You can upgrade the Kubernetes version of a cluster by updating the Kubernetes layer of the [cluster profile](/cluster-profiles) and applying the cluster profile update to the cluster.
+
+</InfoBox>
+
+
+# Manual Certificate Renewal
+
+You can renew the cluster PKI certificates on-demand using the Palette API or the Palette User Interface (UI). When you manually trigger a certificate renewal action, the action will result in cluster nodes becoming repaved. Palette will scale up the cluster control plane nodes count and deploy new nodes with new PKI certificates auto-generated by kubeadm. Once the new control plane nodes are active and available, Palette will remove the previous control plane nodes. The worker nodes renew once the control plane nodes are updated and available.
+
+<br />
+
+<WarningBox>
+
+A manual renewal of cluster PKI certificates will result in all cluster nodes becoming repaved.
+
+</WarningBox>
+
+
+Review the following sections to learn more about manually renewing the cluster PKI certificates. Choose the preferred update method, API, or through the Palette UI.
```

Comment:
```suggestion
Review the following sections to learn more about manually renewing the cluster PKI certificates. Choose your preferred update method, using the Palette UI or the API.
```

Reply:
Optionally, you could use a colon:

"...preferred update method: Palette UI or the API."

I switched the order to match the tab order.